test/transferSpec.js

```diff
@@ -82,6 +82,50 @@ describe('Plugin transfers (optimistic)', function () {
       }, transferA)).catch(done)
     })
 
+    it('should emit a transfer with correct fields with `outgoing_transfer`', function (done) {
+      const id = uuid()
+
+      this.pluginA.once('outgoing_transfer', (transfer) => {
+        console.log(transfer)
+        assert.equal(transfer.id, id)
+        assert.isOk(transfer.amount)
```

Comment:
I would agree that fees are plugin specific. For something like Ripple or Bitcoin I think it would make sense to have a fee parameter in the send function and in the outgoing transfer emitted. Until we come across a case where this really messes things up, I think the `outgoing_transfer` should be the same as the one sent.